### PR TITLE
Additional perl packages and r packages installations

### PR DESCRIPTION
**Pull request by nephantes**

### CODE REVIEW

File: packages/package_r_crisprseek_1_11_0/tool_dependencies.xml [hunk 1]
```diff
@@ -6,6 +6,17 @@
...
+                <action type="setup_perl_environment">
+                    <repository name="package_perl_5_18" owner="iuc">
+                        <package name="perl" version="5.18.1" />
+                    </repository>
+                   <!-- allow downloading and installing an Perl package from cpan.org-->
+                    <package>https://cran.fhcrc.org/src/contrib/futile.logger_1.4.1.tar.gz</package>
+                    <package>https://cran.fhcrc.org/src/contrib/futile.logger_1.4.1.tar.gz</package>
+                    <package>https://cran.r-project.org/src/contrib/snow_0.4-1.tar.gz</package>
+                    <package>https://cran.r-project.org/src/contrib/data.table_1.9.6.tar.gz</package>
+                    <package>https://cran.r-project.org/src/contrib/seqinr_3.1-3.tar.gz</package>
```
Comment (erasche) [inline packages/package_r_crisprseek_1_11_0/tool_dependencies.xml]: cran can be pretty unstable. Would you mind using MRAN? Just pick a date: https://mran.revolutionanalytics.com/snapshot/

File: packages/package_r_crisprseek_1_11_0/tool_dependencies.xml [hunk 2]
```diff
@@ -6,6 +6,17 @@
     <package name="crisprseek" version="1.11.0">
         <install version="1.0">
             <actions>
+                <action type="setup_perl_environment">
+                    <repository name="package_perl_5_18" owner="iuc">
+                        <package name="perl" version="5.18.1" />
+                    </repository>
+                   <!-- allow downloading and installing an Perl package from cpan.org-->
+                    <package>https://cran.fhcrc.org/src/contrib/futile.logger_1.4.1.tar.gz</package>
+                    <package>https://cran.fhcrc.org/src/contrib/futile.logger_1.4.1.tar.gz</package>
```
Comment (bgruening) [inline packages/package_r_crisprseek_1_11_0/tool_dependencies.xml]: This package is given twice.

**Comment by nephantes** [comment]
Are you going to merge this?

**Comment by bgruening** [comment]
@nephantes sure, please give us a little bit of time to test it.

**Comment by bgruening** [comment]
@nephantes this is not working yet.
I have worked a little bit on this package, but I don't have time to test this now.
If you have time please test this one here:
https://github.com/galaxyproject/tools-iuc/compare/pr/373?expand=1

File: packages/package_r_crisprseek_1_11_0/tool_dependencies.xml [hunk 3]
```diff
@@ -6,6 +6,16 @@
...
             <actions>
+                <action type="setup_perl_environment">
+                    <repository name="package_perl_5_18" owner="iuc">
+                        <package name="perl" version="5.18.1" />
+                    </repository>
+                   <!-- allow downloading and installing an Perl package from cpan.org-->
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-11-06/src/contrib/futile.logger_1.4.1.tar.gz</package>
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-11-06/src/contrib/snow_0.4-1.tar.gz</package>
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-11-06/src/contrib/data.table_1.9.6.tar.gz</package>
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-11-06/src/contrib/seqinr_3.1-3.tar.gz</package>
```
Comment (erasche) [inline packages/package_r_crisprseek_1_11_0/tool_dependencies.xml]: also, do these belong in the setup perl section? I think they belong in setup_r_environment instead.

**Comment by nephantes** [comment]
I couldn't test this dependency package for cripsrseek in my own toolshed since, I don't have all dependencies installed. I get error for every single dependency when I used my own toolshed. I was testing it using testtoolshed. When you don't merge it, I am not sure how can try it. Let me know, if you have easy way to test this.
Thanks,
Alper

**Comment by bgruening** [comment]
It seems that you are still missing libraries. This is what I could figure out.

```
                    <!-- bioarchive.github.io -->
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BiocGenerics_0.16.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/IRanges_2.4.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/zlibbioc_1.16.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/XVector_0.10.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Biostrings_2.38.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/ade4_1.7-2.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/seqinr_3.1-3.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomeInfoDb_1.6.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomicRanges_1.22.1.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/XML_3.98-1.3.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/bitops_1.0-6.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/RCurl_1.95-4.7.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/lambda.r_1.1.7.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/futile.options_1.0.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/futile.logger_1.4.1.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/snow_0.4-1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BiocParallel_1.4.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Rsamtools_1.22.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Biobase_2.30.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/SummarizedExperiment_1.0.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomicAlignments_1.6.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/rtracklayer_1.30.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BSgenome_1.38.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/chron_2.3-47.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/data.table_1.9.6.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/CRISPRseek_1.10.0.tar.gz</package>
                </action>
```

**Comment by nephantes** [comment]
Hi Bjorn,
Thanks for your response. I don’t know, what other packages missing. How can I try this dependency package either on my mirror or any other place. I couldn’t create any sandbox environment to test this package. So, I was expecting your help to find out the problems.
Thanks,
Alper


On Nov 14, 2015, at 4:35 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


It seems that you are still missing libraries. This is what I could figure out.

                    
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BiocGenerics_0.16.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/IRanges_2.4.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/zlibbioc_1.16.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/XVector_0.10.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Biostrings_2.38.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/ade4_1.7-2.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/seqinr_3.1-3.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomeInfoDb_1.6.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomicRanges_1.22.1.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/XML_3.98-1.3.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/bitops_1.0-6.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/RCurl_1.95-4.7.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/lambda.r_1.1.7.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/futile.options_1.0.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/futile.logger_1.4.1.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/snow_0.4-1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BiocParallel_1.4.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Rsamtools_1.22.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/Biobase_2.30.0.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/SummarizedExperiment_1.0.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/GenomicAlignments_1.6.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/rtracklayer_1.30.1.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/BSgenome_1.38.0.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/chron_2.3-47.tar.gz</package>
                    <package>https://cran.rstudio.com/src/contrib/data.table_1.9.6.tar.gz</package>
                    <package>https://bioconductor.org/packages/3.2/bioc/src/contrib/CRISPRseek_1.10.0.tar.gz</package>
                </action>

**Comment by bgruening** [comment]
@nephantes have a look at my last post, these are all R packages you need to install crisprseek.
I use this script to generate such files and all dependencies: https://github.com/bioarchive/aRchive_source_code/pull/20

**Comment by nephantes** [comment]
Hi Bjorn,
I don’t use nginx, my server is apache and where should be this directory to build the archive for apache server?
"/srv/nginx/bioarchive.galaxyproject.org/root/<http://bioarchive.galaxyproject.org/root/%E2%80%9C>“<http://bioarchive.galaxyproject.org/root/%E2%80%9C> and how can I enable it in my toolshed?
Thanks,
Alper

On Nov 14, 2015, at 5:49 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


@nephantes<https://github.com/nephantes> have a look at my last post, these are all R packages you need to install crisprseek.
I use this script to generate such files and all dependencies: bioarchive/aRchive_source_code#20<https://github.com/bioarchive/aRchive_source_code/pull/20>

**Comment by nephantes** [comment]
Hi Bjorn,
I run your script and it gave the packages below and those are different than the ones I saw in the errors. How can I try, if these are going to work or not for CRISPRseek?

                    <package>https://bioarchive.galaxyproject.org/HSMMSingleCell_0.102.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/BiocGenerics_0.14.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/Biobase_2.28.0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/Rcpp_0.12.1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/plyr_1.8.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/digest_0.6.8.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/gtable_0.1.2.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/stringi_1.0-1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/magrittr_1.5.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/stringr_1.0.0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/reshape2_1.4.1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/RColorBrewer_1.1-2.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/dichromat_2.0-0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/colorspace_1.2-6.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/munsell_0.4.2.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/labeling_0.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/scales_0.3.0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/proto_0.3-10.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/MASS_7.3-45.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/ggplot2_1.0.1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/VGAM_1.0-0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/lattice_0.20-33.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/Matrix_1.2-2.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/registry_0.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/codetools_0.2-14.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/xtable_1.8-0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/pkgmaker_0.22.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/rngtools_1.2.4.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/cluster_2.0.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/gridBase_0.4-7.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/iterators_1.0.8.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/foreach_1.4.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/doParallel_1.0.10.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/NMF_0.20.6.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/irlba_2.0.0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/igraph_1.0.1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/combinat_0.0-8.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/fastICA_1.2-0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/matrixStats_0.15.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/limma_3.24.15.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/monocle_1.2.0.tar.gz</package>

On Nov 14, 2015, at 5:49 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


@nephantes<https://github.com/nephantes> have a look at my last post, these are all R packages you need to install crisprseek.
I use this script to generate such files and all dependencies: bioarchive/aRchive_source_code#20<https://github.com/bioarchive/aRchive_source_code/pull/20>

**Comment by bgruening** [comment]
Hi,

you need to change the package name in this script. Currently it generates all dependencies for monocle.
As soon as you have it upload it to the Test Tool Shed and install it into Galaxy. It should work.

**Comment by nephantes** [comment]
Perfect! I got this list. I hope, it is complete.

                    <package>https://bioarchive.galaxyproject.org/BiocGenerics_0.14.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/S4Vectors_0.6.6.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/IRanges_2.2.9.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/XVector_0.8.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/zlibbioc_1.14.0.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/Biostrings_2.36.4.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/GenomeInfoDb_1.4.3.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/GenomicRanges_1.20.8.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/XML_3.98-1.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/bitops_1.0-6.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/RCurl_1.95-4.7.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/Rsamtools_1.20.5.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/lambda.r_1.1.7.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/futile.options_1.0.0.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/futile.logger_1.4.1.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/snow_0.4-1.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/BiocParallel_1.2.22.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/GenomicAlignments_1.4.2.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/rtracklayer_1.28.10.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/BSgenome_1.36.3.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/ade4_1.7-2.tar.gz</package>
                    <package>http://cran.rstudio.com/src/contrib/seqinr_3.1-3.tar.gz</package>
                    <package>https://bioarchive.galaxyproject.org/CRISPRseek_1.8.1.tar.gz</package><https://bioarchive.galaxyproject.org/CRISPRseek_1.8.1.tar.gz%3C/package%3E>

On Nov 14, 2015, at 9:17 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,

you need to change the package name in this script. Currently it generates all dependencies for monocle.
As soon as you have it upload it to the Test Tool Shed and install it into Galaxy. It should work.

**Comment by nephantes** [comment]
Could you please update the repository on testtoolshed? I am not the owner and I cannot test it on testtoolshed without updating the latest commit.
Thanks,
Alper

On Nov 14, 2015, at 9:28 PM, Alper Kucukural <alper.kucukural@umassmed.edu<mailto:alper.kucukural@umassmed.edu>> wrote:

Perfect! I got this list. I hope, it is complete.

                    <package>https://bioarchive.galaxyproject.org/BiocGenerics_0.14.0.tar.gz</package><https://bioarchive.galaxyproject.org/BiocGenerics_0.14.0.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/S4Vectors_0.6.6.tar.gz</package><https://bioarchive.galaxyproject.org/S4Vectors_0.6.6.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/IRanges_2.2.9.tar.gz</package><https://bioarchive.galaxyproject.org/IRanges_2.2.9.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/XVector_0.8.0.tar.gz</package><https://bioarchive.galaxyproject.org/XVector_0.8.0.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/zlibbioc_1.14.0.tar.gz</package><https://bioarchive.galaxyproject.org/zlibbioc_1.14.0.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/Biostrings_2.36.4.tar.gz</package><https://bioarchive.galaxyproject.org/Biostrings_2.36.4.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/GenomeInfoDb_1.4.3.tar.gz</package><https://bioarchive.galaxyproject.org/GenomeInfoDb_1.4.3.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/GenomicRanges_1.20.8.tar.gz</package><https://bioarchive.galaxyproject.org/GenomicRanges_1.20.8.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/XML_3.98-1.3.tar.gz</package><http://cran.rstudio.com/src/contrib/XML_3.98-1.3.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/bitops_1.0-6.tar.gz</package><http://cran.rstudio.com/src/contrib/bitops_1.0-6.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/RCurl_1.95-4.7.tar.gz</package><http://cran.rstudio.com/src/contrib/RCurl_1.95-4.7.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/Rsamtools_1.20.5.tar.gz</package><https://bioarchive.galaxyproject.org/Rsamtools_1.20.5.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/lambda.r_1.1.7.tar.gz</package><http://cran.rstudio.com/src/contrib/lambda.r_1.1.7.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/futile.options_1.0.0.tar.gz</package><http://cran.rstudio.com/src/contrib/futile.options_1.0.0.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/futile.logger_1.4.1.tar.gz</package><http://cran.rstudio.com/src/contrib/futile.logger_1.4.1.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/snow_0.4-1.tar.gz</package><http://cran.rstudio.com/src/contrib/snow_0.4-1.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/BiocParallel_1.2.22.tar.gz</package><https://bioarchive.galaxyproject.org/BiocParallel_1.2.22.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/GenomicAlignments_1.4.2.tar.gz</package><https://bioarchive.galaxyproject.org/GenomicAlignments_1.4.2.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/rtracklayer_1.28.10.tar.gz</package><https://bioarchive.galaxyproject.org/rtracklayer_1.28.10.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/BSgenome_1.36.3.tar.gz</package><https://bioarchive.galaxyproject.org/BSgenome_1.36.3.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/ade4_1.7-2.tar.gz</package><http://cran.rstudio.com/src/contrib/ade4_1.7-2.tar.gz%3C/package%3E>
                    <package>http://cran.rstudio.com/src/contrib/seqinr_3.1-3.tar.gz</package><http://cran.rstudio.com/src/contrib/seqinr_3.1-3.tar.gz%3C/package%3E>
                    <package>https://bioarchive.galaxyproject.org/CRISPRseek_1.8.1.tar.gz</package><https://bioarchive.galaxyproject.org/CRISPRseek_1.8.1.tar.gz%3C/package%3E>

On Nov 14, 2015, at 9:17 PM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,

you need to change the package name in this script. Currently it generates all dependencies for monocle.
As soon as you have it upload it to the Test Tool Shed and install it into Galaxy. It should work.

**Comment by bgruening** [comment]
This is superseded by #390 right?

**Comment by nephantes** [comment]
Yes true.

On Nov 17, 2015, at 10:35 AM, Björn Grüning <notifications@github.com<mailto:notifications@github.com>> wrote:


This is superseded by #390<https://github.com/galaxyproject/tools-iuc/pull/390> right?

**Pull request closed without merging**